Lombiq.Tests.UI/SecurityScanning/AutomationFrameworkPlans/Baseline.yml

```diff
@@ -42,8 +42,14 @@ jobs:
     enableTags: false
     disableAllRules: false
   rules:
-  - id: 10035
-    name: "Strict-Transport-Security Header"
```

Comment:
This needs to be disabled because during UI testing, because a local app needn't use HSTS. It's also something commonly configured outside the app, like in Cloudflare.

Reply:
I see. Added the explanation as comment.